monkey/infection_monkey/exploit/HostExploiter.py

```diff
@@ -104,20 +104,13 @@ def exploit_host(
 
     def pre_exploit(self):
         self.exploit_result = ExploiterResultData(
-            os=self.host.os.get("type"), info=self.exploit_info, attempts=self.exploit_attempts
+            os=self.host.os.get("type"), info=self.exploit_info
         )
         self.set_start_time()
 
     def _is_interrupted(self):
         return self.interrupt.is_set()
 
-    def _set_interrupted(self):
-        # This method should be refactored to raise an exception to reduce duplication in the
-        # "if is_interrupted: return self.exploitation_results"
-        # Ideally the user should only do "check_for_interrupt()"
-        logger.info("Exploiter has been interrupted")
-        self.exploit_result.interrupted = True
-
```

Comment:
See https://github.com/guardicore/monkey/commit/b0f03179c1058797b85f9a3f5ea9e32f9fe61482. Maybe we want to leave it?

Reply:
Everything uses `_is_interrupted`. We could leave it, but I don't know of any upcoming changes where we would need this. Most of the time it doesn't matter if exploiter was interrupted or not, our baseline now is events. If we want to indicate that, just send `ExploiterInterrupted` event or something. It's not part of results